### PR TITLE
Remove Unnecessary Whitespace

### PR DESCRIPTION
**Pull request by alexthemitchell**
* It's my pet peeve

### CODE REVIEW

File: Lyles/Lyles/BeerMenuTableViewController.m [hunk 1]
```diff
@@ -27,7 +27,4 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
   return cell;
 }
 
-
-
```
Comment (alexthemitchell) [inline Lyles/Lyles/BeerMenuTableViewController.m:31]: This comment is a comment

Comment (penglinq) [inline Lyles/Lyles/BeerMenuTableViewController.m:31]: Whatever...

File: Lyles/Lyles/BeerMenuTableViewController.m [hunk 2]
```diff
@@ -27,7 +27,4 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
   return cell;
 }
 
-
```
Comment (hardtmad) [inline Lyles/Lyles/BeerMenuTableViewController.m:30]: It's my pet peeve too!

**Review by hardtmad — APPROVED** [review]
Nice work!

**Review by enathang — CHANGES_REQUESTED** [review]
Stop being so anal.

**Review by penglinq — APPROVED** [review]

**Pull request merged**